### PR TITLE
Userrights changes for WikiCanada

### PR DESCRIPTION
**Pull request by Amanda1234567**

### CODE REVIEW

**Comment by Reception123** [comment]
Staff is more okay as a name but why not just leave it at Steward?

**Comment by Amanda1234567** [comment]
@Reception123 because when I see "steward" I think of only global CentralAuth actions. I want one group that has all restricted rights, and "staff" is a less misleading name

**Comment by JohnFLewis** [comment]
Stewards can only add groups that exist on metawiki.

**Comment by Amanda1234567** [comment]
@JohnFLewis I thought that stewards can do userrights-interwiki from Meta and add any/all groups that exist on the local wiki specified

**Comment by JohnFLewis** [comment]
No, they have to exist on meta to be assigned. It's a MediaWiki bug that's been waiting years for a fix.

**Comment by Amanda1234567** [comment]
Okay, so is it possible to configure settings so that the "founder" group can add/remove the "staff" group to all users **except** themselves?

(i.e. If a steward needs rights from the staff group, I would add the group to their account locally, but I wouldn't be able to assign the right to myself)

**Comment by JohnFLewis** [comment]
Not a possibility in MediaWiki.

**Comment by Amanda1234567** [comment]
@JohnFLewis I've changed it back to steward

File: LocalWiki.php [hunk 1]
```diff
@@ -195,6 +194,7 @@ function onClosedSiteNoticeAfter( &$siteNotice, $skin ) {
 	unset( $wgGroupPermissions['oversight'] );
 	unset( $wgGroupPermissions['autoreview'] );
 	unset( $wgGroupPermissions['editor'] );
+	unset( $wgGroupPermissions['steward'] );
```
Comment (JohnFLewis) [inline LocalWiki.php]: needs to be removed

**Comment by Amanda1234567** [comment]
@JohnFLewis done

**Pull request merged**